src/guides/ethereum-overview/index.md

```diff
@@ -100,32 +116,46 @@ Here's the whole process visually:
 
 **Ethereum is a blockchain that allows you to run programs in its trusted environment.** This contrasts with the Bitcoin blockchain, which only allows you to manage cryptocurrency.
 
-To this end, Ethereum has a virtual machine, called the Ethereum Virtual Machine (EVM). The EVM allows code to be verified and executed on the blockchain, providing guarantees it will be run the same way on everyone's machine. This code is contained in "smart contracts" (more on these below).
+In computer science terms, Ethereum is a "Turing complete", virtual machine. Simply put, Ethereum is an internet-based computer - A single symbolic computer that runs on thousands of physical computers. This virtual computer is called the Ethereum Virtual Machine (EVM). The EVM allows code to be verified and executed on the blockchain, providing guarantees it will be run the same way on everyone's machine. This code is contained in "smart contracts" (more on these below).
```

Comment:
I don't get what a single symbolic computer is :/ Also, not everyone will know what Turing complete is, and we should give a little more explanation?